### PR TITLE
added sort button functionality and fixed some styling

### PR DESCRIPTION
**Pull request by wcampbell91**

### CODE REVIEW

**Review by nathanieltylerowens — APPROVED** [review]
👍

**Review by MLogan1998 — CHANGES_REQUESTED** [review]
change index.html to shop.html

File: main.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ const sweatpants = [
     gender: "Kids",
     price: 15, 
     img: "https://cdn.shoplightspeed.com/shops/613188/files/9352569/600x600x1/chaser-kids-chaser-kids-girls-love-cozy-sweatpants.jpg",
-    description: "Perfect for inside or outside play time, these sweats are guaranteed to keep your child moving to his or her's hearts content!"
+    description: "These sweaty's sweats are perfect for inside or outside play time, these sweats will keep your child moving to his or her heart's content!"
```
Comment (petestewart) [inline main.js]: Sweaty's should be capitalized

File: main.js [hunk 2]
```diff
@@ -111,6 +111,36 @@ const buildProductCards = (sweatpants) => {
   printToDom('#productCardContainer', domString)
 }
 
+const filterSweats = (event) => {
+  let buttonId = event.target.id;
+  const tempSweatsArr = [];
+
+  console.log(buttonId)
```
Comment (petestewart) [inline main.js]: console.log still in function

File: shop.html [hunk 3]
```diff
@@ -13,7 +13,7 @@ <h1>Sweaty's</h1>
   <p>The Best Sweatpants of All Time</p>
   <nav>
     <ul id="navLinks">
-      <li><a href="shop.html" id="navToShop">Shop</a></li>
+      <li><a href="index.html" id="navToShop">Shop</a></li>
```
Comment (petestewart) [inline shop.html]: should be shop.html

**Review by petestewart — CHANGES_REQUESTED** [review]
Capitalization, console log and index.html link should be fixed

**Review by petestewart — APPROVED** [review]

**Review by MLogan1998 — APPROVED** [review]
looks good

**Pull request merged**